### PR TITLE
documentation(documentation-website): Update API bench link on docs w…

### LINKED ISSUE #958: Update api-bench link on docs website

### PR DESCRIPTION
**Pull request by arnabnandy7**
# The Pull Request is ready

- [x] fixes #958 
- [ ] all actions are passing
- [x] only fixes a single issue

## Documentation-Website

- [ ] mobile view is usable
- [ ] desktop view is usable
- [ ] no a-tags are used directly (NavLink, MailLink, ExternalLink instead)
- [ ] all new texts are added to the translation files (at least the english one)
- [ ] tests have been added (if required)
- [ ] shared code has been extracted in a different file

### CODE REVIEW

File: documentation-website/cypress/e2e/autowiring.cy.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@
     cy.url().should('eq', BASE_URL + page.autowiring,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Autowiring | Usage | @idrinth/api-bench',);
+    cy.title().should('eq', 'Autowiring | Usage | @idrinth-api-bench/api-bench',);
```
[the remark this answers is not in the document]
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/autowiring.cy.ts]: It's resolved

File: documentation-website/cypress/e2e/contributors.cy.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@
   },);
   it('title matches', () => {
     cy.title()
-      .should('eq', 'Contributors | Contributing | @idrinth/api-bench',);
+      .should('eq', 'Contributors | Contributing | @idrinth-api-bench/api-bench',);
```
[the remark this answers is not in the document]
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/contributors.cy.ts]: It's resolved

File: documentation-website/src/components/header.tsx [hunk 3]
```diff
@@ -32,27 +32,27 @@
...
-          'https://hub.docker.com/r/idrinth/api-bench-gitlab-runner',
-          'https://hub.docker.com/r/idrinth/api-bench-gitea-action',
-          'https://hub.docker.com/r/idrinth/api-bench',
-          'https://hub.docker.com/r/idrinth/api-bench-history-microservice',
-          'https://hub.docker.com/r/idrinth/api-bench-history-website',
+          'https://hub.docker.com/r/idrinth-api-bench/api-bench-gitlab-runner',
+          'https://hub.docker.com/r/idrinth-api-bench/api-bench-gitea-action',
+          'https://hub.docker.com/r/idrinth-api-bench/api-bench',
+          'https://hub.docker.com/r/idrinth-api-bench/api-bench-history-microservice',
+          'https://hub.docker.com/r/idrinth-api-bench/api-bench-history-website',
```
[the remark this answers is not in the document]
Comment (arnabnandy7) [reply documentation-website/src/components/header.tsx]: It's resolved

File: documentation-website/cypress/e2e/middlewares.cy.ts [hunk 4]
```diff
@@ -11,7 +11,7 @@
     cy.url().should('eq', BASE_URL + page.middlewares,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Middlewares | Usage | @idrinth/api-bench',);
+    cy.title().should('eq', 'Middlewares | Usage | @idrinth-api-bench/api-bench',);
```
[the remark this answers is not in the document]
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/middlewares.cy.ts]: It's resolved

File: documentation-website/cypress/e2e/sponsors.cy.ts [hunk 5]
```diff
@@ -11,7 +11,7 @@
     cy.url().should('eq', BASE_URL + page.sponsors,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Sponsors | Contributing | @idrinth/api-bench',);
+    cy.title().should('eq', 'Sponsors | Contributing | @idrinth-api-bench/api-bench',);
```
[the remark this answers is not in the document]
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/sponsors.cy.ts]: It's resolved

File: documentation-website/language/it.yml [hunk 6]
```diff
@@ -19,54 +19,54 @@ nav:
...
+    title1: "idrinth-api-bench/api-bench-gitea-action su DockerHub"
     text1: "Gitea Action"
-    title2: "idrinth/api-bench su DockerHub"
+    title2: "idrinth-api-bench/api-bench su DockerHub"
     text2: "Pure Alpine"
-    title3: "idrinth/api-bench-history-microservice su DockerHub"
+    title3: "idrinth-api-bench/api-bench-history-microservice su DockerHub"
     text3: "History Microservice"
-    title4: "idrinth/api-bench-history-website su DockerHub"
+    title4: "idrinth-api-bench/api-bench-history-website su DockerHub"
```
[the remark this answers is not in the document]
Comment (arnabnandy7) [reply documentation-website/language/it.yml]: It's resolved

File: documentation-website/cypress/e2e/middlewares.cy.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/middlewares.cy.ts]: Issue resolved

File: documentation-website/cypress/e2e/autowiring.cy.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/autowiring.cy.ts]: Issue resolved

File: documentation-website/cypress/e2e/sponsors.cy.ts (hunk 5 above)
[the remark this answers is not in the document]
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/sponsors.cy.ts]: Issue resolved

**Comment by arnabnandy7** [comment]
@coderabbitai review

**Comment by arnabnandy7** [comment]
@coderabbitai review

**Review by Idrinth — CHANGES_REQUESTED** [review]
Please remove all changes that are unrelated to the new github url, they don't make sense yet, due to the packages not yet existing

**Comment by arnabnandy7** [comment]
> Please remove all changes that are unrelated to the new github url, they don't make sense yet, due to the packages not yet existing

Hi @Idrinth I've already removed changes I made in package json files. Any other files you're pointing?

**Comment by Idrinth** [comment]
> > Please remove all changes that are unrelated to the new github url, they don't make sense yet, due to the packages not yet existing
> 
> Hi @Idrinth I've already removed changes I made in package json files. Any other files you're pointing?

Great! May be in readmes for package names as well - it's pretty much ONLY about the github link so far, since everything else needs to be published under the new name first

File: documentation-website/src/components/header.tsx [hunk 7]
```diff
@@ -81,7 +81,7 @@ const Header = ({
         label={'slack'}
       />
       <SocialLink
-        to="https://github.com/Idrinth/api-bench"
+        to="https://github.com/idrinth-api-bench/api-bench"
```
Comment (marikadeveloper) [inline documentation-website/src/components/header.tsx]: you can keep this

File: documentation-website/src/pages/contributing/index.tsx [hunk 8]
```diff
@@ -79,7 +79,7 @@ const Index = () => <Layout
         <Lang lnkey={'contributing.code.tests.manual.description'}/>
       </p>
       <Code language='bash'>{`
-git clone https://github.com/idrinth/api-bench
+git clone https://github.com/idrinth-api-bench/api-bench
```
Comment (marikadeveloper) [inline documentation-website/src/pages/contributing/index.tsx:82]: you can keep this

File: documentation-website/src/pages/contributing/index.tsx [hunk 9]
```diff
@@ -118,7 +118,7 @@ npm run coai
         <Lang lnkey={'contributing.code.setup.description'}/>
       </p>
       <Code language='bash'>{`
-git clone https://github.com/idrinth/api-bench
+git clone https://github.com/idrinth-api-bench/api-bench
```
Comment (marikadeveloper) [inline documentation-website/src/pages/contributing/index.tsx:121]: you can keep this

**Comment by marikadeveloper** [comment]
> > Please remove all changes that are unrelated to the new github url, they don't make sense yet, due to the packages not yet existing
> 
> Hi @Idrinth I've already removed changes I made in package json files. Any other files you're pointing?

Hi @arnabnandy7 you should only keep **changes related to the new github repo url**, the npm package is still https://www.npmjs.com/package/@idrinth/api-bench it's not changed yet. So no need to change the stuff in the localization files or the website's header yet :) I left you some comments of what you can keep.

**Comment by arnabnandy7** [comment]
> > > Please remove all changes that are unrelated to the new github url, they don't make sense yet, due to the packages not yet existing
> > 
> > Hi @Idrinth I've already removed changes I made in package json files. Any other files you're pointing?
> 
> Hi @arnabnandy7 you should only keep **changes related to the new github repo url**, the npm package is still https://www.npmjs.com/package/@idrinth/api-bench it's not changed yet. So no need to change the stuff in the localization files or the website's header yet :) I left you some comments of what you can keep.

That's great @marikadeveloper thanks for your support. I'll make the changes.

File: documentation-website/src/pages/contributing/index.tsx (hunk 8 above)
Comment (arnabnandy7) [reply documentation-website/src/pages/contributing/index.tsx:82]: sure

File: documentation-website/src/pages/contributing/index.tsx (hunk 9 above)
Comment (arnabnandy7) [reply documentation-website/src/pages/contributing/index.tsx:121]: sure

File: documentation-website/src/components/header.tsx (hunk 7 above)
Comment (arnabnandy7) [reply documentation-website/src/components/header.tsx]: sure

**Comment by arnabnandy7** [comment]
@Idrinth @marikadeveloper it's done, please review once

File: documentation-website/src/pages/usage/routes/index.tsx [hunk 10]
```diff
@@ -22,7 +22,7 @@ const Route = () => <Layout
       </p>
       <Code language='typescript'>{`import {
   Task,
-} from '@idrinth/api-bench';
+} from '@idrinth-api-bench/api-bench';
```
Comment (Idrinth) [inline documentation-website/src/pages/usage/routes/index.tsx]: this package will never exist, please revert the change for now

File: documentation-website/src/pages/quick-start/index.tsx [hunk 11]
```diff
@@ -18,7 +18,7 @@ const Index = () => <Layout
     prefix={'quick-start.step2.description'}
     postfix={'quick-start.step2.result'}
     level={'h2'}
-    language={'bash'}>npx -p @idrinth/api-bench-cli iabmp</CodeUnit>
+    language={'bash'}>npx -p @idrinth-api-bench/api-bench-cli iabmp</CodeUnit>
```
Comment (Idrinth) [inline documentation-website/src/pages/quick-start/index.tsx]: this package will never exist, please revert the change for now

File: documentation-website/src/components/header.tsx [hunk 12]
```diff
@@ -81,7 +81,7 @@ const Header = ({
         label={'slack'}
       />
       <SocialLink
-        to="https://github.com/Idrinth/api-bench"
+        to="https://github.com/idrinth/api-bench"
```
Comment (Idrinth) [inline documentation-website/src/components/header.tsx]: This is one of the links that actually has to be updated

File: documentation-website/src/components/default-meta.tsx [hunk 13]
```diff
@@ -28,7 +28,7 @@ const DefaultMeta = ({
         <Head.Meta
           content={description}
           name='description'/>
-        <Head.Title>{title} | @idrinth/api-bench</Head.Title>
+        <Head.Title>{title} | @idrinth-api-bench/api-bench</Head.Title>
```
Comment (Idrinth) [inline documentation-website/src/components/default-meta.tsx]: this package will never exist, please revert the change for now

File: documentation-website/language/it.yml [hunk 14]
```diff
@@ -43,7 +43,7 @@ socials:
     title: "idrinth/api-bench su Medium"
     alt: "Medium"
   youtube:
-    title: "idrinth-api-bench su YouTube"
+    title: "idrinth su YouTube"
```
Comment (Idrinth) [inline documentation-website/language/it.yml]: the youtube name is already the one on the left, please revert the change

File: documentation-website/index.html [hunk 15]
```diff
@@ -4,8 +4,8 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="./src/assets/favicon.svg"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta name="description" content="A documentation site for the open-source benchmarking framework @idrinth/api-bench">
-    <title>@idrinth/api-bench</title>
+    <meta name="description" content="A documentation site for the open-source benchmarking framework @idrinth-api-bench/api-bench">
+    <title>@idrinth-api-bench/api-bench</title>
```
Comment (Idrinth) [inline documentation-website/index.html]: this package will never exist, please revert the change for now

File: documentation-website/index.html [hunk 16]
```diff
@@ -4,8 +4,8 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="./src/assets/favicon.svg"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta name="description" content="A documentation site for the open-source benchmarking framework @idrinth/api-bench">
-    <title>@idrinth/api-bench</title>
+    <meta name="description" content="A documentation site for the open-source benchmarking framework @idrinth-api-bench/api-bench">
```
Comment (Idrinth) [inline documentation-website/index.html]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/usage.cy.ts [hunk 17]
```diff
@@ -11,7 +11,7 @@ describe('usage', () => {
     cy.url().should('eq', BASE_URL + page.usage,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Usage | @idrinth/api-bench',);
+    cy.title().should('eq', 'Usage | @idrinth-api-bench/api-bench',);
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/usage.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/support.cy.ts [hunk 18]
```diff
@@ -11,7 +11,7 @@ describe('support', () => {
     cy.url().should('eq', BASE_URL + page.support,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Support | @idrinth/api-bench',);
+    cy.title().should('eq', 'Support | @idrinth-api-bench/api-bench',);
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/support.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/storage.cy.ts [hunk 19]
```diff
@@ -11,7 +11,7 @@ describe('usage/storage', () => {
     cy.url().should('eq', BASE_URL + page.storage,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Storage | Usage | @idrinth/api-bench',);
+    cy.title().should('eq', 'Storage | Usage | @idrinth-api-bench/api-bench',);
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/storage.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/sponsors.cy.ts [hunk 20]
```diff
@@ -11,7 +11,8 @@ describe('contributing/sponsors', () => {
     cy.url().should('eq', BASE_URL + page.sponsors,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Sponsors | Contributing | @idrinth/api-bench',);
+    cy.title().should('eq', 'Sponsors | Contributing | '+
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/sponsors.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/routes.cy.ts [hunk 21]
```diff
@@ -11,7 +11,7 @@ describe('usage/route', () => {
     cy.url().should('eq', BASE_URL + page.routes,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Routes | Usage | @idrinth/api-bench',);
+    cy.title().should('eq', 'Routes | Usage | @idrinth-api-bench/api-bench',);
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/routes.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/results.cy.ts [hunk 22]
```diff
@@ -11,7 +11,7 @@ describe('usage/results', () => {
     cy.url().should('eq', BASE_URL + page.results,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Results | Usage | @idrinth/api-bench',);
+    cy.title().should('eq', 'Results | Usage | @idrinth-api-bench/api-bench',);
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/results.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/quick-start.cy.ts [hunk 23]
```diff
@@ -11,12 +11,12 @@ describe('usage/quick-start', () => {
...
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Quick Start | @idrinth/api-bench',);
+    cy.title().should('eq', 'Quick Start | @idrinth-api-bench/api-bench',);
   },);
   it('h1 matches', () => {
     cy.get('h1',)
       .invoke('text',)
-      .should('eq', '@idrinth/api-bench quick start',);
+      .should('eq', '@idrinth-api-bench/api-bench quick start',);
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/quick-start.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/quick-start.cy.ts [hunk 24]
```diff
@@ -11,12 +11,12 @@ describe('usage/quick-start', () => {
     cy.url().should('eq', BASE_URL + page.quickStart,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Quick Start | @idrinth/api-bench',);
+    cy.title().should('eq', 'Quick Start | @idrinth-api-bench/api-bench',);
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/quick-start.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/not-found.cy.ts [hunk 25]
```diff
@@ -11,7 +11,7 @@ describe('404', () => {
     cy.url().should('eq', BASE_URL + page.notFound,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', '404 NOT FOUND | @idrinth/api-bench',);
+    cy.title().should('eq', '404 NOT FOUND | @idrinth-api-bench/api-bench',);
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/not-found.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/middlewares.cy.ts [hunk 26]
```diff
@@ -11,7 +11,8 @@ describe('usage/middlewares', () => {
     cy.url().should('eq', BASE_URL + page.middlewares,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Middlewares | Usage | @idrinth/api-bench',);
+    cy.title().should('eq', 'Middlewares | Usage | ' +
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/middlewares.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/logging.cy.ts [hunk 27]
```diff
@@ -11,7 +11,7 @@ describe('usage/logging', () => {
     cy.url().should('eq', BASE_URL + page.logging,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Logging | Usage | @idrinth/api-bench',);
+    cy.title().should('eq', 'Logging | Usage | @idrinth-api-bench/api-bench',);
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/logging.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/license.cy.ts [hunk 28]
```diff
@@ -11,7 +11,7 @@ describe('license', () => {
     cy.url().should('eq', BASE_URL + page.license,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'License | @idrinth/api-bench',);
+    cy.title().should('eq', 'License | @idrinth-api-bench/api-bench',);
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/license.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/language-switch.cy.ts [hunk 29]
```diff
@@ -21,7 +21,7 @@ describe('language-switch', () => {
   },);
 
   it('changes language correctly', () => {
-    const pageSignature = '@idrinth/api-bench';
+    const pageSignature = '@idrinth-api-bench/api-bench';
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/language-switch.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/imprint.cy.ts [hunk 30]
```diff
@@ -11,7 +11,7 @@ describe('imprint', () => {
     cy.url().should('eq', BASE_URL + page.imprint,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Imprint | @idrinth/api-bench',);
+    cy.title().should('eq', 'Imprint | @idrinth-api-bench/api-bench',);
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/imprint.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/home.cy.ts [hunk 31]
```diff
@@ -11,7 +11,7 @@ describe('home', () => {
     cy.url().should('eq', BASE_URL + page.home,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Home | @idrinth/api-bench',);
+    cy.title().should('eq', 'Home | @idrinth-api-bench/api-bench',);
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/home.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/contributors.cy.ts [hunk 32]
```diff
@@ -12,7 +12,8 @@ describe('contributors', () => {
   },);
   it('title matches', () => {
     cy.title()
-      .should('eq', 'Contributors | Contributing | @idrinth/api-bench',);
+      .should('eq', 'Contributors | Contributing | '+
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/contributors.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/contributing.cy.ts [hunk 33]
```diff
@@ -11,7 +11,7 @@ describe('contributing', () => {
     cy.url().should('eq', BASE_URL + page.contributing,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Contributing | @idrinth/api-bench',);
+    cy.title().should('eq', 'Contributing | @idrinth-api-bench/api-bench',);
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/contributing.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/code-of-conduct.cy.ts [hunk 34]
```diff
@@ -11,7 +11,7 @@ describe('home', () => {
     cy.url().should('eq', BASE_URL + page['code-of-conduct'],);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Code of Conduct | @idrinth/api-bench',);
+    cy.title().should('eq', 'Code of Conduct | @idrinth-api-bench/api-bench',);
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/code-of-conduct.cy.ts]: this package will never exist, please revert the change for now

File: documentation-website/cypress/e2e/autowiring.cy.ts [hunk 35]
```diff
@@ -11,7 +11,8 @@ describe('usage/autowiring', () => {
     cy.url().should('eq', BASE_URL + page.autowiring,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Autowiring | Usage | @idrinth/api-bench',);
+    cy.title().should('eq', 'Autowiring | Usage | '
```
Comment (Idrinth) [inline documentation-website/cypress/e2e/autowiring.cy.ts]: this package will never exist, please revert the change for now

**Review by Idrinth — CHANGES_REQUESTED** [review]

File: documentation-website/src/components/header.tsx (hunk 12 above)
Comment (arnabnandy7) [reply documentation-website/src/components/header.tsx]: Done

File: documentation-website/src/pages/usage/routes/index.tsx (hunk 10 above)
Comment (arnabnandy7) [reply documentation-website/src/pages/usage/routes/index.tsx]: Reverted

File: documentation-website/src/pages/quick-start/index.tsx (hunk 11 above)
Comment (arnabnandy7) [reply documentation-website/src/pages/quick-start/index.tsx]: Reverted

File: documentation-website/src/components/default-meta.tsx (hunk 13 above)
Comment (arnabnandy7) [reply documentation-website/src/components/default-meta.tsx]: Reverted

File: documentation-website/language/it.yml (hunk 14 above)
Comment (arnabnandy7) [reply documentation-website/language/it.yml]: Reverted

File: documentation-website/index.html (hunk 16 above)
Comment (arnabnandy7) [reply documentation-website/index.html]: Reverted

Comment (arnabnandy7) [reply documentation-website/index.html]: Reverted

File: documentation-website/index.html (hunk 15 above)
Comment (arnabnandy7) [reply documentation-website/index.html]: Reverted

File: documentation-website/cypress/e2e/usage.cy.ts (hunk 17 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/usage.cy.ts]: Reverted

File: documentation-website/cypress/e2e/support.cy.ts (hunk 18 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/support.cy.ts]: Reverted

File: documentation-website/cypress/e2e/storage.cy.ts (hunk 19 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/storage.cy.ts]: Reverted

File: documentation-website/cypress/e2e/sponsors.cy.ts (hunk 20 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/sponsors.cy.ts]: Reverted

File: documentation-website/cypress/e2e/routes.cy.ts (hunk 21 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/routes.cy.ts]: Reverted

File: documentation-website/cypress/e2e/results.cy.ts (hunk 22 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/results.cy.ts]: Reverted

File: documentation-website/cypress/e2e/quick-start.cy.ts (hunk 24 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/quick-start.cy.ts]: Reverted

File: documentation-website/cypress/e2e/quick-start.cy.ts (hunk 23 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/quick-start.cy.ts]: Reverted

File: documentation-website/cypress/e2e/middlewares.cy.ts [hunk 36]
```diff
@@ -11,7 +11,8 @@
...
```
[the remark this answers is not in the document]
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/middlewares.cy.ts]: Done

File: documentation-website/cypress/e2e/middlewares.cy.ts (hunk 26 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/middlewares.cy.ts]: Reverted

File: documentation-website/cypress/e2e/logging.cy.ts (hunk 27 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/logging.cy.ts]: Reverted

File: documentation-website/cypress/e2e/license.cy.ts (hunk 28 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/license.cy.ts]: Reverted

File: documentation-website/cypress/e2e/language-switch.cy.ts (hunk 29 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/language-switch.cy.ts]: Reverted

File: documentation-website/cypress/e2e/imprint.cy.ts (hunk 30 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/imprint.cy.ts]: Reverted

File: documentation-website/cypress/e2e/home.cy.ts (hunk 31 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/home.cy.ts]: Reverted

File: documentation-website/cypress/e2e/contributors.cy.ts (hunk 32 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/contributors.cy.ts]: Reverted

File: documentation-website/cypress/e2e/contributing.cy.ts (hunk 33 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/contributing.cy.ts]: Reverted

File: documentation-website/cypress/e2e/code-of-conduct.cy.ts (hunk 34 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/code-of-conduct.cy.ts]: Reverted

File: documentation-website/cypress/e2e/autowiring.cy.ts (hunk 35 above)
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/autowiring.cy.ts]: Reverted

File: documentation-website/cypress/e2e/middlewares.cy.ts (hunk 36 above)
[the remark this answers is not in the document]
Comment (arnabnandy7) [reply documentation-website/cypress/e2e/middlewares.cy.ts]: Fixed

**Comment by arnabnandy7** [comment]
@Idrinth it's done, please let me know if more to be done.
